roles/confluent.variables/vars/main.yml

```
@@ -475,6 +475,12 @@ kafka_connect:
 
 kafka_connect_http_protocol: "{{ 'https' if kafka_connect_ssl_enabled|bool else 'http' }}"
 
+kafka_connect_rest_extension_classes:
+  - "{% if rbac_enabled|bool %}io.confluent.connect.security.ConnectSecurityExtension{% endif %}"
+  - "{% if kafka_connect_secret_registry_enabled|bool %},io.confluent.connect.secretregistry.ConnectSecretRegistryExtension{% endif %}"
+ 
+kafka_connect_final_rest_extension_classes: "{{kafka_connect_rest_extension_classes| difference(['']) + kafka_connect_custom_rest_extension_classes.split(',')}}"
```

```suggestion
kafka_connect_final_rest_extension_classes: "{{ (kafka_connect_rest_extension_classes | difference(['']) + kafka_connect_custom_rest_extension_classes.split(',')) | unique}}"
```

We should probably add the unique filter just to be safe